src/glyph/tests.rs

```diff
@@ -639,6 +639,22 @@ fn unexpected_line_after_offcurve2() {
     let _ = parse_glyph(data.as_bytes()).unwrap();
 }
 
+#[test]
+#[should_panic(expected = "InvalidName")]
+fn invalid_name() {
+    let data = "
```

Comment:
You could use `r#"..."#` so you don't have to escape stuff inside :)

Reply:
Unfortunately this test case requires including a control character in the name, which has to be escaped, and so we can't use a raw string. 😢